<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Verbs Writer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="bRun">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bTest">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Test</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bTest1">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Test1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbLangs">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>English</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>French</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>German</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Italian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Latin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Spanish</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Folder:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leFolderPath">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Start Num:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leStartNum">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QPlainTextEdit" name="pteOut">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>File:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leFilePath">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>one-line.txt</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>leFilePath</tabstop>
  <tabstop>leFolderPath</tabstop>
  <tabstop>leStartNum</tabstop>
  <tabstop>bRun</tabstop>
  <tabstop>bTest</tabstop>
  <tabstop>bTest1</tabstop>
  <tabstop>cbLangs</tabstop>
  <tabstop>pteOut</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
